### DISCUSSION TITLE
Cannot reverify as my college is not showing up in the list nor in the maps

### DISCUSSION THREAD

**Discussion by AyushShahh** [community/community#51381] [GitHub Education]
My college name is not in the list and also not on the map. What should I do?
I had applied a month ago and it was there and my application was approved.

![image](https://user-images.githubusercontent.com/71313814/228620840-b384d18d-8d61-4338-8ff4-d6aad8b67115.png)

**Comment by F0XN0T** [comment]
I understand that you're experiencing challenges with verifying your college for the GitHub Student Developer Pack, and it's concerning to hear that you haven't received a satisfactory response from GitHub. It's important for platforms to provide personalized attention and support to their users while handling personal information with care and respect user privacy.

If you're still encountering difficulties, you could try contacting GitHub's customer support team and explaining your situation in detail. Additionally, you could try reaching out to other students and creators to see if they have any suggestions for resolving this issue.

I hope that a resolution is found promptly as it's affecting many users and requires urgent attention.

**Reply by AyushShahh** [reply]
GitHub's education support team has not replied to me yet
